armi/utils/gridEditor.py

```diff
@@ -1051,9 +1051,15 @@ def setNumRings(self, n: int):
         Change the number of rings that should be drawn
         """
         self.numRings = n
-        if self.grid.geomType == geometry.HEX:
+        if (
+            self.grid.geomType == geometry.HEX
+            or self.grid.geomType == geometry.GeomType.HEX
+        ):
             grid = grids.HexGrid.fromPitch(1, numRings=self.numRings)
-        elif self.grid.geomType == geometry.CARTESIAN:
+        elif (
+            self.grid.geomType == geometry.CARTESIAN
+            or self.grid.geomType == geometry.GeomType.CARTESIAN
+        ):
```

Comment:
Bumped into this in the course of my work today, fixing in #236; apologies for the conflict to come

Reply:
No prob!